cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -713,6 +713,13 @@ public List<Service> generateExternalBootstrapServices() {
                 }
             }
 
+            if (KafkaListenerType.NODEPORT == listener.getType()) {
+                List<String> eips = ListenersUtils.bootstrapExternalIPs(listener);
```

Comment:
I would go for a more exhaustive variable name. Something like `externalIps` instead of just `eips`.